text/0000-lts-release-strategy.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Fabric LTS release strategy
+nav_order: 3
+---
+
+- Feature Name: Fabric LTS release strategy
+- Start Date: 2020-03-18
+- RFC PR:
+- Fabric Component:
+- Fabric Issue:
+
+# Summary
+[summary]: #summary
+
+Define Long-term support (LTS) release strategy for Hyperledger Fabric
+
+Long-term support definition from wikipedia.org:
+
+> **Long-term support (LTS)** is a product lifecycle management policy in which a stable release of computer software is maintained for a longer period of time than the standard edition.
+
+> **LTS** applies the tenets of reliability engineering to the software development process and software release life cycle. Long-term support extends the period of software maintenance; it also alters the type and frequency of software updates (patches) to reduce the risk, expense, and disruption of software deployment, while promoting the dependability of the software.
+
+# Motivation
+[motivation]: #motivation
+
+Administrators of production deployments find LTS releases desirable as they provide confidence that a certain software release will be maintained for an extended period of time.
+
+Specifically, for Hyperledger Fabric, administrators can expect the following of a LTS release:
+
+- Stable and well-tested code
+- API contract stability
+- 3rd digit patch releases for bug and security fixes, delivered for an extended period of time on a regular cadence (or more frequently as needed)
+- Minimal set of feature additions and other changes, reducing the risk of functional regressions and bugs
+
```

Comment:
.. that can be applied via an upgrade process that does not require data migration.

Reply:
simplified as "that can easily be applied"